README.md

```
@@ -67,7 +69,7 @@ DOCLING_URL='http://localhost:8004/v1'
 WHISPER_URL='http://localhost:50001/v1'
 ```
 
-> **Note:** The `HF_AUTH_TOKEN` is required for Hugging Face API access and model downloads; you can create a token [here](https://huggingface.co/settings/tokens). Set `LLM_REASONING=True` to enable extended reasoning/thinking in the LLM responses.
+> **Note:** The `HUGGING_FACE_HUB_TOKEN` is required for Hugging Face API access and model downloads; you can create a token [here](https://huggingface.co/settings/tokens). Set `LLM_REASONING=true` to enable extended reasoning/thinking in the LLM responses.
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)